tools/parse_baselines/main.py

```diff
@@ -104,8 +117,17 @@ def main():
     # Instantiate the right data parser.
     parser = DATA_PARSERS[args.test](data_provider)
 
-    # Finally, parse and print the baselines.
-    print(json.dumps(parser.parse(), indent=4))
+    # Finally, parse and update the baselines.
+    with open(f"./tests/integration_tests/performance/configs/"
+              f"test_{args.test}_config_{args.kernel}.json",
+              'r+', encoding="utf8") as baselines_file:
+        json_baselines = json.load(baselines_file)
+        cpus = parser.parse()
+        json_baselines["hosts"]["instances"][args.instance] = {"cpus": cpus}
+
+        baselines_file.truncate(0)
+        baselines_file.seek(0, 0)
+        json.dump(json_baselines, baselines_file, indent=4)
```

Comment:
How about something like this?

```python
baselines_file = Path(f"tests/integration_tests/performance/configs/test_{args.test}_config_{args.kernel}.json")
json_baselines = json.load(baselines_file.read_text())
...
baselines_file.write_text(json_baselines)
```

Reply:
Thanks for the comment and sorry I missed it.

Seems more appropriate to use `Path()`, indeed. But for the last part personally I would keep `json.dump()` instead of `write_text()` in order to be able to specify the indent and maybe other json-specific parameters in the future.